pdr_backend/sim/sim_engine.py

```diff
@@ -328,12 +331,19 @@ def _sell(self, price: float, tokcoin_amt_send: float) -> float:
 
         return usdcoin_amt_recd
 
+    def disable_realtime_state(self):
+        self.state_updates = False
+
     @enforce_types
     def save_state(self, i: int, N: int):
         "Save state on this iteration Y/N?"
```

Comment:
is this comment a TODO? If so, please mark it as such. I know that pylint breaks, but that's the point. We should not merge anything that has questions pending.

Reply:
@calina-c Not sure what you refer to, I don't see any comment.